resources/platform/linux/peach/pits/README.md

```diff
@@ -0,0 +1,75 @@
+Adding a pit:
```

Comment:
nit: add `#` in the beginning so that this gets rendered as a title

Reply:
Done